doc/manual/methods.rst

```diff
@@ -226,148 +235,29 @@ values is perhaps the single most powerful and central feature of the
 Julia language. Core operations typically have dozens of methods:
 
 .. doctest::
+   :options: +SKIP
```

Comment:
I might take this out right before release and see how annoying it gets while backporting fixes